odoo/addons/base/models/res_partner.py

```diff
@@ -516,6 +524,11 @@ def _fields_sync(self, values):
 
         # 2. To DOWNSTREAM: sync children
         self._children_sync(values)
+        """
+        synchronize the Salesperson(user_id) if Salesperson is not set on the parent
```

Comment:
That's not really what I meant.

The docstring of **the method** should be modified with this part added to it.